### DISCUSSION TITLE
Floating bar

### DISCUSSION THREAD

**Discussion by dzmitryabramenia** [quasarframework/quasar#13252] [General - Components / Directives / etc]
Help me find the code how to make such a floating bar. I'm guessing it's done via q-scroll-area, but I'd like to see some sample code.
![Screenshot (94)](https://user-images.githubusercontent.com/85438900/164996005-85c1839b-bc12-4beb-8afa-d96081472385.png)

**Comment by yusufkandemir** [comment]
It's a [QDrawer](https://quasar.dev/layout/drawer) with a [QScrollArea](https://quasar.dev/vue-components/scroll-area) inside, which contains a [QList](https://quasar.dev/vue-components/list-and-list-items#qlist-api) with bunch of [QItem](https://quasar.dev/vue-components/list-and-list-items#qitem-api)s in it.
https://github.com/quasarframework/quasar/blob/f0de80bfea7580358487629a0db8f9abfece2a39/docs/src/layouts/DocLayout.vue#L96-L119
